core/templating/template_helpers.go

```diff
@@ -398,6 +496,56 @@ func (t templateHelpers) csvCountRows(dataSourceName string) string {
 	return fmt.Sprintf("%d", numRows)
 }
 
+func (t templateHelpers) csvSQL(queryString string) []RowMap {
```

Comment:
go prefer camel case, so `csvSqlQuery` is more idiomatic

Reply:
I will change

Reply:
Done